dashboard/.gitattributes

```diff
@@ -1,3 +0,0 @@
-yarn.lock binary
```

Comment:
nit - we might want to keep this around?

Reply:
We should. It was dropped when I used the next.js docs to create a new repo and then dropped the old code and shoved the new code in its place. I will revert this.